src/appkit.cs

```diff
@@ -11336,6 +11606,12 @@ partial interface NSPasteboard // NSPasteboard does _not_ implement NSPasteboard
 		[Field ("NSPasteboardNameDrag")]
 		NSString NSPasteboardNameDrag { get; }
 
+		[Field ("kUTTypeFileURL")]
+		NSString UTTypeFileURL { get; }
+
+		[Field ("kUTTypeVCard")]
+		NSString UTTypeVCard { get; }
```

Comment:
`UT*` constants are generally in another framework

Reply:
Good catch. Let me dig into this.

Reply:
They're in mobilecoreservices.cs